conda_build/build.py

```diff
@@ -582,7 +582,7 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
         else:
             test_file = join(tmp_dir, 'run_test.sh')
             # TODO: Run the test/commands here instead of in run_test.py
-            cmd = ['/bin/bash', '-x', '-e', test_file]
+            cmd = ['/bin/sh', '-x', '-e', test_file]
```

Comment:
I think it should be fine.  Conda build should be shell agnostic.  Enforce bash should, theoretically, be done in the build script (because the build script knows if it is using a specific shell feature or not).